Tasks/Common/docker-common/containerimageutils.ts

```diff
@@ -26,6 +27,11 @@ export function generateValidImageName(imageName: string): string {
     return imageName;
 }
 
+export function getBaseImageNameFromDockerFileContent(dockerFilePath: string): string {
```

Comment:
rename the method to getBaseImageNameFromDockerFile

Reply:
done